instalint-http-daemon/src/main/java/io/instalint/daemon/AnalyzerServlet.java

```diff
@@ -11,7 +11,7 @@
 
 public class AnalyzerServlet extends HttpServlet {
 
-  private Backend backend;
+  private static Backend backend;
```

Comment:
I would guess, that this is a good idea. Is this really safe to change? (Taking redeployments, etc. into account)

Reply:
The rule description of squid:S2226 says this when the field is not `static`:

> By contract, a servlet container creates one instance of each servlet and then a dedicated thread is attached to each new incoming HTTP request to process this request. So all threads are sharing the servlet instances and by extension instance fields. To prevent any misunderstanding and unexpected behavior at runtime, all servlet fields should then be either static and/or final, or simply removed.

If there is only ever one instance, then this is safe. But it's ugly. I think we should improve this at some point.

Reply:
Sorry I missed that.